tests/libvmi/vmi.go

```diff
@@ -125,6 +125,13 @@ func WithSEV() Option {
 	}
 }
 
+// WithTPM adds a virtual TPM device.
```

Comment:
@jean-edouard I can't find the place where this function is being used, is there a chance its a leftover?

Reply:
You are correct, it is not currently in use, but I thought I'd leave it there for future use.
After all, this is technically called a lib :)
Will remove on request.

Reply:
I would probably don't introduce it as you alone are not using it.

Reply:
Removed